### ISSUE TITLE
Thousands of `HTTP 204` only solved by service restart

### ISSUE THREAD

**Issue by solracsf** [nextcloud/notify_push#96]
These days we were having a high server load and we couldn't really understand why.
After looking at the `nginx` access logs, we've found thousands and thousansds (even in the rotated logs) of lines like this:

`"DELETE /remote.php/dav/files/user/hiers/page/RPB-04-4M.zip HTTP/1.1" 204 0 "-" "Mozilla/5.0 (Windows) mirall/3.2.0stable-Win64 (build 20210409) (Nextcloud, wiecture: x86_64 OsArchitecture: x86_64)"`

Please note the [204 status](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/204). And we've searched from where these could come from. Well, after issue a `systemctl restart notify-push` all of those 204 have stopped, and the server load had become to normal levels again some minutes after.

But some days later, same issue...and same solution (restart the `systemd` service).

**Bottom line**: this app has some bug, somewhere, and we can't run it on production on long runnings, and restarts seems to be needed regularly.

Let me know if we can help debug this.

Thanks.
